manuscript/manuscript.Rmd

```diff
@@ -30,130 +30,150 @@ bibliography: references.bib
 header-includes:
   - \usepackage{amsmath}
   - \usepackage{amssymb}
-
 output: 
   rticles::arxiv_article:
     keep_tex: true
+editor_options: 
+  markdown: 
+    wrap: sentence
```

Comment:
ooh that's cool, haven't tried the visual editor before!

Does look like it mucks a bit with formatting below, mostly equivalent changes, though not sure why it seems to switch non-parenthetical references like @someone to parenthetical [@someone].

Reply:
Ah weird! I just accepted the default wrap settings and didn't realize it made real rather than just visual formatting changes. But ya, I love the visual editor, so much easier to read.